### PR TITLE
Add new iPad Air (3th generation) and iPad Mini (5th generation).

### PR DESCRIPTION
**Pull request by Zandor300**
Add the new iPad Air (3th generation) and iPad mini (5th generation) to DeviceKit.

This is still work in progress since the device model identifiers haven't been released yet and without them, the devices can't be discovered.

https://www.theiphonewiki.com/wiki/Models

@dennisweissmann

### CODE REVIEW

**Comment by Zandor300** [comment]
The `allDevicesWithSensorHousing` and `hasSensorHousing` functions were removed because they weren't added through the gyb file. (https://github.com/dennisweissmann/DeviceKit/pull/168 fixes this)

**Comment by dennisweissmann** [comment]
Thanks! :) I just wanted to open this issue since it also came up here: 
https://stackoverflow.com/questions/26028918/how-to-determine-the-current-iphone-device-model/26962452?noredirect=1#comment97349201_26962452

File: Source/Device.swift.gyb [hunk 1]
```diff
@@ -61,14 +61,14 @@ iPads = [
...
             Device("iPad5",           "Device is an [iPad 5](https://support.apple.com/kb/SP751)",                      "https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/SP751/ipad_5th_generation.png",                  ["iPad6,11", "iPad6,12"],                     9.7,  (3, 4),   "iPad 5", 264, False, False, False, False, True, False),
             Device("iPad6",           "Device is an [iPad 6](https://support.apple.com/kb/NotYetAvailable)",            "https://support.apple.com/library/APPLE/APPLECARE_ALLGEOS/SP751/ipad_5th_generation.png",                  ["iPad7,5", "iPad7,6"],                       9.7,  (3, 4),   "iPad 6", 264, False, False, False, False, True, False),
+            Device("iPadAir3",        "Device is an [iPad Air (3th generation)](INSERT LINK)",                          "https://store.storeimages.cdn-apple.com/4982/as-images.apple.com/is/image/AppleInc/aos/published/images/i/pa/ipad/mini/ipad-mini-compare-201903", ["iPad11,3", "iPad11,4"],       10.5, (3, 4),   "iPad Air (3th generation)", 264, False, False, False, False, True, False),
```
Comment (dennisweissmann) [inline Source/Device.swift.gyb]: Nitpick: 3rd generation

Comment (Zandor300) [reply Source/Device.swift.gyb]: Yeah, `.iPad3` is called iPad (3th generation) so I think we should leave it like this?

Comment (dennisweissmann) [reply Source/Device.swift.gyb]: Sorry for being unclear 🙈 I meant the text (comment) “Device is an [iPad Air 3**rd** generation)]”, not “3**th** generation.

Comment (Zandor300) [reply Source/Device.swift.gyb]: Ah yes, I will fix that 🤣

**Comment by Zandor300** [comment]
@dennisweissmann Ah, well, I won't be able to merge again since @devicekit-danger-bot starts complaining when there is a merge commit in the pull request. I don't think there is a different way to update the branch from master without having a merge commit. I've tried everything to rebase but can't get it to work.

You may merge it if you feel like it.

**Comment by dennisweissmann** [comment]
Sorry for the trouble, not sure why rebasing is not working for you. I cannot look into it currently, so I’ll just merge it :) thanks 🙏

**Comment by Zandor300** [comment]
@dennisweissmann @denisenepraunig Maybe one of you could release a new version to release support for iPad Air (3rd generation) and iPad Mini (5th generation)?

**Comment by denisenepraunig** [comment]
Hello @Zandor300 - yes I will prepare a new 1.13.0 version so that all the latest changes (everything that is currently in master - including the iPad Air 3rd gen and iPad Mini 5th) will be released.

**Comment by denisenepraunig** [comment]
Hi @Zandor300. I made a version 1.13.0 with commit 66130e8 and published the new version to CocoaPods. Yay! :)

**Pull request merged**